anchor/network/src/network.rs

```diff
@@ -94,10 +108,40 @@ fn build_anchor_behaviour(local_keypair: Keypair) -> AnchorBehaviour {
         identify::Behaviour::new(identify_config)
     };
 
+    // TODO those values might need to be parameterized based on the network
+    let slots_per_epoch = 32;
+    let seconds_per_slot = 12;
+    let duplicate_cache_time = Duration::from_secs(slots_per_epoch * seconds_per_slot); // 6.4 min
+
+    let gossip_message_id = move |message: &gossipsub::Message| {
+        gossipsub::MessageId::from(&Sha256::digest(&message.data)[..20])
```

Comment:
we should also have the [network related](https://github.com/sigp/lighthouse/blob/0d90135047519f4c2ee586d50e560f7bb2ff9b10/beacon_node/lighthouse_network/src/config.rs#L444-L467) prefixes right?

Reply:
I believe the spec doesn't mention it https://github.com/ssvlabs/ssv-spec/blob/main/p2p/CONFIG.md#msg-id